examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "GroundTruth:    cat  ship  ship plane\n",
+      "Predicted:    cat  ship  ship  ship\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataiter = iter(testloader)\n",
+    "images, labels = dataiter.next()\n",
+    "\n",
+    "# print images\n",
+    "imshow(torchvision.utils.make_grid(images))\n",
+    "print('GroundTruth: ', ' '.join('%5s' % classes[labels[j]] for j in range(4)))\n",
+    "\n",
+    "outputs = net(images.to(device))\n",
+    "\n",
+    "_, predicted = torch.max(outputs, 1)\n",
+    "\n",
+    "print('Predicted: ', ' '.join('%5s' % classes[predicted[j]]\n",
+    "                              for j in range(4)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's look at the accuracy over test set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "cuda:0\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(device)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy of the network on the 10000 test images: 52 %\n"
```

Comment:
The accuracy looks very low, should we train more epochs?

Reply:
Yes. changed to more epochs.